test/integration/src/py/test_utilities.py

```diff
@@ -21,12 +26,13 @@ class EthereumToSifchainTransferRequest:
     sifchain_fees: str = ""
     smart_contracts_dir: str = ""
     ethereum_chain_id: str = "5777"
-    chain_id: str = "localnet"
+    chain_id: str = "localnet" #cosmos chain id
     manual_block_advance: bool = True
-    n_wait_blocks: int = 4
+    n_wait_blocks: int = n_wait_blocks
     bridgebank_address: str = ""
     bridgetoken_address: str = ""
     sifnodecli_node: str = "tcp://localhost:26657"
```

Comment:
This needs to be parameterized as well to get balances from sifchain on testnet

Reply:
Right now, we just have a constant n_wait_blocks set in test_utilities.py.  Since there's no way to get it at runtime (I think?) I'm not sure what else we should do.  This change just uses the constant that everyone else uses; it was incorrectly hardcoded to 4 in two places.